akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/headers.scala

```diff
@@ -9,12 +9,13 @@ import java.net.InetSocketAddress
 import java.security.MessageDigest
 import java.util
 import javax.net.ssl.SSLSession
+import akka.stream.scaladsl.ScalaSessionAPI
```

Comment:
This is like a `.toScala` method that also has the language in the name, users won’t use this type much, I reckon (because we provide all the niceness in different ways already).